### PR TITLE
Adding rdp-tools

### PR DESCRIPTION
**Pull request by colinbrislawn**
Adding support for the RDP Tools.
This is my first contribution to this repo, so I would appreciate any feedback @bioconda/core 
Thank you all!

### CODE REVIEW

File: recipes/rdp-tools/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+#    - brew update              # [osx]
+#    - brew install ant         # [osx]
+
+    - git config --global url."https://".insteadOf git://
+    - git submodule init
+    - git submodule update
+
+    - make
+
+    - mkdir -p $PREFIX/bin
```
Comment (bgruening) [inline recipes/rdp-tools/meta.yaml]: This is usually handled in a build.sh file. Please have a look at similar recipes.

File: recipes/rdp-tools/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    - ant
+
+build:
+  number: 0
+  script:
+#    - brew update              # [osx]
+#    - brew install ant         # [osx]
+
+    - git config --global url."https://".insteadOf git://
+    - git submodule init
```
Comment (bgruening) [inline recipes/rdp-tools/meta.yaml]: I don't think this is needd if you use git_url. Recommended is a tarball if there is one packed with all submodules included.

File: recipes/rdp-tools/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+    - make
+
+    - mkdir -p $PREFIX/bin
+    - mv *.jar $PREFIX/bin
+    - mv lib $PREFIX/bin
+
+test:
+  commands:
+    - java -Xmx1g -jar $PREFIX/bin/classifier.jar classify -c 0.5 -o usga_classified.txt -h soil_hier.txt $SRC_DIR/classifier/samplefiles/USGA_2_4_B_trimmed.fasta
```
Comment (bgruening) [inline recipes/rdp-tools/meta.yaml]: Java programs are shipped with a special wrapper that makes is more easily to call from the commandline line. Moreover, the $PREFIX is not available during runtime, afaik.
Please have a look at: https://github.com/bioconda/bioconda-recipes/tree/master/recipes/opsin/2.1.0 for a nice example.

**Comment by bgruening** [comment]
@colinbrislawn thanks for your contribution!

File: recipes/rdp-tools/meta.yaml (hunk 2 above)
Comment (colinbrislawn) [inline recipes/rdp-tools/meta.yaml]: Unfortunately there is not. In fact, this github repo refers to several other repos which include the dependencies. It's kind of a mess.

File: recipes/rdp-tools/meta.yaml (hunk 3 above)
Comment (colinbrislawn) [inline recipes/rdp-tools/meta.yaml]: Unfortunately, RDP tools does not have this kind of wrapper. 😞  This is their example command: https://github.com/rdpstaff/classifier/blob/master/README#L84 

I'll see if $PREFIX works or if there is a suitable replacement.

File: recipes/rdp-tools/meta.yaml (hunk 2 above)
Comment (colinbrislawn) [inline recipes/rdp-tools/meta.yaml]: Good catch!

**Comment by colinbrislawn** [comment]
Works on OSX. On linux
`/anaconda/conda-bld/test-tmp_dir/run_test.sh: line 3: java: command not found`

Any ideas? Should I list java as a runtime dependency?

**Comment by druvus** [comment]
@colinbrislawn It was recently added to bioconda as rdptools.

**Pull request closed without merging**